### ISSUE TITLE
Health metrics recommend contacting explat if there is no data

### ISSUE THREAD

**Issue by jessie-ross** [Automattic/abacus#831]
From p1674552499885689-slack-C7HH3V5AS

When there is no data our health metrics recommend contacting ExPlat.

### To do

- [ ] Don't show the offending health metrics if there is no data.

**Comment by razvanpapadopol** [comment]
Yes, maybe we should consider a more tailored message in the case of runtime > 1 day and no data.

But it should still communicate a similar message. In this case, contacting ExPlat helped figuring out that a wrong experiment name was used in code: p1674568845245219/1674552499.885689-slack-C7HH3V5AS
